src/redux/books/books.js

```diff
@@ -0,0 +1,15 @@
+const addBook = () => ({ type: 'ADD' });
+const removeBook = () => ({ type: 'REMOVE' });
+const initialState = [];
+
+const updateBooks = (state = initialState, action) => {
+  switch (action.type) {
+    case 'ADD':
+      return [...state];
+    default:
+      return state;
```

Comment:
- [ ] You did not define state change for removing a book, Please make sure that you have a case described, responsible for removing a book.
 - You can consider using this code `case 'REMOVE': return state.filter((book) => book.id !== action.payload.id);`